2025-06-13 16:34:09 INFO uvicorn.error Started server process [18928]
2025-06-13 16:34:10 INFO uvicorn.error Waiting for application startup.
2025-06-13 16:34:10 INFO uvicorn.error Application startup complete.
2025-06-13 16:34:35 INFO uvicorn.access 127.0.0.1:49189 - "POST /api/status_refresh_ob_op HTTP/1.1" 422
2025-06-13 16:34:58 ERROR root An unexpected error occurred.
Traceback (most recent call last):
  File "E:\python\kompas_importer_backend\decorators\response.py", line 19, in wrapper
    result = await func(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\python\kompas_importer_backend\routers\api_router\api\office.py", line 45, in status_refresh_ob_op
    async with cls_factory as instance:
  File "C:\Program Files\Python312\Lib\contextlib.py", line 210, in __aenter__
    return await anext(self.gen)
           ^^^^^^^^^^^^^^^^^^^^^
  File "E:\python\kompas_importer_backend\callers\import_generic.py", line 23, in _dependency
    if isinstance(items, list) and issubclass(body_model, BaseModel):
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen abc>", line 123, in __subclasscheck__
TypeError: issubclass() arg 1 must be a class
2025-06-13 16:34:58 INFO uvicorn.access 127.0.0.1:49315 - "POST /api/status_refresh_ob_op HTTP/1.1" 200
2025-06-13 16:36:46 INFO uvicorn.error Shutting down
2025-06-13 16:36:46 INFO uvicorn.error Waiting for application shutdown.
2025-06-13 16:36:46 INFO uvicorn.error Application shutdown complete.
2025-06-13 16:36:46 INFO uvicorn.error Finished server process [18928]
